### PR TITLE
adding containers concept page

### PR DESCRIPTION
**Pull request by jleon15**
## Description

- Adding token containers concept page

## Testing required outside of automated testing?

- [x] Not Applicable

### Screenshots (if appropriate):
image

- [ ] Not Applicable

## Rollback / Rollforward Procedure

- [x] Roll Forward
- [ ] Roll Back

## Reviewer Checklist

- [ ] Description of Change
- [ ] Description of outside testing if applicable.
- [ ] Description of Roll Forward / Backward Procedure
- [ ] Documentation updated for Change

### CODE REVIEW

File: collections/concepts/what-are-containers.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+image:
+path: https://cdn.basistheory.com/images/seo/guides-opengraph.png
+width: 1200
+height: 630
+---
+
+# What are Token Containers?
+
+Token Containers are hierarchical grouping constructs that allow you to logically segment tokens within your tenant. With
+this grouping in place, you are then able to control access to your tokens based on [Access Rules](https://developers.basistheory.com/concepts/access-controls/)
```
Comment (brigonzalez) [inline collections/concepts/what-are-containers.md:18]:
```suggestion
this grouping in place, you are able to control access to your tokens based on [Access Rules](https://developers.basistheory.com/concepts/access-controls/) 
```

File: collections/concepts/what-are-containers.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+Token Containers are hierarchical grouping constructs that allow you to logically segment tokens within your tenant. With
+this grouping in place, you are then able to control access to your tokens based on [Access Rules](https://developers.basistheory.com/concepts/access-controls/) 
+tied to applications that specify which containers an application has access to. 
+
+A container is represented by a hierarchical path, which is conceptually similar to directories in a UNIX filesystem.
+Container names must start and end with a `/`, and the root container is denoted by `/`. Container names may include any 
+alphanumeric characters, `-`, or `_`, and can contain an arbitrary number of nested sub-containers.
+
+You have complete control to customize your container hierarchy to meet your unique data governance requirements. By default,
+tokens will be assigned to containers based off of their data classification and impact levels, using the format
```
Comment (brigonzalez) [inline collections/concepts/what-are-containers.md:26]:
```suggestion
tokens will be assigned to containers based off their data classification and impact levels, using the format 
```

**Review by brigonzalez — COMMENTED** [review]
just two small comments. and same as before with capitalizing "application". we should be consistent with that

File: collections/concepts/what-are-containers.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+Token Containers are hierarchical grouping constructs that allow you to logically segment tokens within your tenant. With
+this grouping in place, you are able to control access to your tokens based on [Access Rules](https://developers.basistheory.com/concepts/access-controls/) 
+tied to applications that specify which containers an Application has access to. 
+
+A container is represented by a hierarchical path, which is conceptually similar to directories in a UNIX filesystem.
+Container names must start and end with a `/`, and the root container is denoted by `/`. Container names may include any 
+alphanumeric characters, `-`, or `_`, and can contain an arbitrary number of nested sub-containers.
+
+You have complete control to customize your container hierarchy to meet your unique data governance requirements. By default,
+tokens will be assigned to containers based off their data classification and impact levels, using the format
```
Comment (brigonzalez) [inline collections/concepts/what-are-containers.md:26]:
```suggestion
Tokens will be assigned to containers based off their data classification and impact levels, using the format 
```

File: collections/concepts/what-are-containers.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+tokens will be assigned to containers based off their data classification and impact levels, using the format 
+`/<classification>/<impact_level>/`, e.g. `/pci/high/` or `/general/low/`.
+
+---
+
+## Common Use Cases
+
+### Segmenting by data classification
+
+If you need to segment tokens by data classification, you could organize your tokens into the following containers:
```
Comment (brigonzalez) [inline collections/concepts/what-are-containers.md:35]:
```suggestion
If you need to segment Tokens by data classification, you could organize your Tokens into the following containers:
```

File: collections/concepts/what-are-containers.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+- `/bank/`
+- `/pii/`
+- `/general/`
+- `/my-custom-classification/`
+
+---
+
+### Segmenting by customer
+
+If you need to segment tokens by customer, you could organize your tokens into the following containers:
```
Comment (brigonzalez) [inline collections/concepts/what-are-containers.md:47]:
```suggestion
If you need to segment Tokens by customer, you could organize your tokens into the following containers:
```

File: collections/concepts/what-are-containers.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+path: https://cdn.basistheory.com/images/seo/guides-opengraph.png
+width: 1200
+height: 630
+---
+
+# What are Token Containers?
+
+Token Containers are hierarchical grouping constructs that allow you to logically segment tokens within your tenant. With
+this grouping in place, you are able to control access to your tokens based on [Access Rules](https://developers.basistheory.com/concepts/access-controls/) 
+tied to applications that specify which containers an Application has access to.
```
[the remark this answers is not in the document]
Comment (jleon15) [reply collections/concepts/what-are-containers.md:19]: i don't really like to capitalize the plural form of the entity, if it was Token or Application it makes sense but for plural I think we should use lowercase

[the remark this answers is not in the document]
Comment (brigonzalez) [reply collections/concepts/what-are-containers.md:19]: what's your reasoning for that? I still see them as resources that we capitalize

Comment (dhudec) [inline collections/concepts/what-are-containers.md:19]: The wording here is confusing and overly complex.

```suggestion
Tokens can be logically grouped into Containers to segment access to tokens within a Tenant. Fine-grained access control policies can be defined by attaching one or more [Access Rules](https://developers.basistheory.com/concepts/access-controls/) to an Application. Each Access Rule is scoped to a specific Container of tokens, allowing you to grant an Application permission to interact with only a subset of tokens within your Tenant.
```

**Review by dhudec — APPROVED** [review]

**Pull request merged**